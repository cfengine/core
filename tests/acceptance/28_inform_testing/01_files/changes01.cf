body file control
{
  inputs => { "$(sys.policy_entry_dirname)/../common.cf.sub" };
}

bundle common testcase
{
  vars:
    "filename" string => "$(this.promise_filename)";
}

bundle agent setup
{
  files:
    "$(G.testfile)"
      create => "true";
}

bundle agent main
{
  files:
    "$(G.testfile)"
      changes => example;
}

body changes example
{
  hash => "best";
  update_hashes => "true";
  report_changes => "content";
}
