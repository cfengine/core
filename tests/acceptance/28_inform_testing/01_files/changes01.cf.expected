    info: Stored md5 hash for '/tmp/TEST.cfengine' (MD5=d41d8cd98f00b204e9800998ecf8427e)
    info: Stored sha1 hash for '/tmp/TEST.cfengine' (SHA=da39a3ee5e6b4b0d3255bfef95601890afd80709)
    info: files promise '/tmp/TEST.cfengine' repaired
